code/controllers/subsystems/mapping.dm

```diff
@@ -141,9 +141,10 @@ SUBSYSTEM_DEF(mapping)
 // Z-Level procs after this point.
 /datum/controller/subsystem/mapping/proc/get_gps_level_name(var/z)
 	if(z)
-		var/obj/abstract/level_data/level = levels_by_z[z]
-		if(level?.name)
-			return level.get_gps_level_name()
+		var/datum/level_data/level = levels_by_z[z]
+		. = level?.get_display_name()
```

Comment:
Is it possible that gps/overmap tries to get names before exoplanets may have generated? Because, that would be my guess.
But, yeah, I'll change that to runtime if level is null.